### PR TITLE
Send tag create and push webhook when release created on UI

### LINKED ISSUE #5288: Drone is not recognizing gitea release creation
- Gitea version (or commit ref): 1.5.3
- Git version:
- Operating system: RHEL
- Database (use `[x]`):
  - [X] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [X] Not relevant

## Description
As mentioned [here](https://github.com/drone/drone/issues/2519) and [here](https://github.com/go-gitea/gitea/issues/3738) for me, there seems to be some issues with gitea webhooks.

if i create a release from the gitea ui (creates also a tag in background), drone does not recognize it and is not running the pipeline.

If i instead push a tag from command line to remote repo, drone will run twice. One run for the push and one run for the tag... That duplicates the runtime. Is this the expected behavior?

### PR DESCRIPTION
**Pull request by blueworrybear**
This pull request is to fix #5288 and #6027

This update will send both tag `create` and `push` event after release created.
Since the release creation creates a new tag in git repository, it seems to me that it makes sense to send tag `create` and `push` event.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -75,6 +75,8 @@ require (
 	github.com/mattn/go-sqlite3 v1.11.0
 	github.com/mcuadros/go-version v0.0.0-20190308113854-92cdf37c5b75
 	github.com/microcosm-cc/bluemonday v0.0.0-20161012083705-f77f16ffc87a
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```
Comment (lunny) [inline go.mod]: Please use go1.13

**Comment by lunny** [comment]
I think push should be handler by post-receive.

File: go.mod (hunk 1 above)
Comment (blueworrybear) [reply go.mod]: Okay, thank you. I'll update my GO version.

**Comment by blueworrybear** [comment]
> I think push should be handler by post-receive.

I'll take a look if I could handle the webhook on post-receive.

**Comment by blueworrybear** [comment]
@lunny 
I have update to go V1.13. But I'm not sure if push could be handled by post-receive.
The `tag` for release is created directly in the remote repository. Therefore there is no `git push` during release creation actually.

Therefore, I would like to keep the codes. If there is any misunderstanding, please let me know.
Thanks for your help. 😄

**Comment by lunny** [comment]
I think there is a wrong base branch on this PR.

**Comment by blueworrybear** [comment]
@lunny 

Humm... it's weird... I'll merge with master in my local workspace and pushagain.

**Comment by blueworrybear** [comment]
I think I mess up my branch... I'll close this PR and recreate a new one. Sorry about that 😭

**Pull request closed without merging**